data/map.txt

```diff
@@ -29808,11 +29808,11 @@ planet Zug
 		threshold 3000
 		fleet "Large Militia" 18
 
-planet wormhole_barren
+planet "Wormhole Barren Alpha"
 	attributes "requires: quantum keystone"
 	description ``
-planet wormhole_barren2
+planet "Wormhole Barren Beta"
 	attributes "requires: quantum keystone"
 	description ``
-planet wormhole_link
+planet "Wormhole Link"
 	description ``
```

Comment:
Red wormholes should mean that a stone is required for landing, yet this wormhole doesn't require one despite being red.

Reply:
Thanks. Will fix this.